config.yml

```diff
@@ -250,6 +250,22 @@ scoutsuite:
       - pipenv --version
       - scout --version
 
+serverless:
+  "1.48":
+    build_args:
+      PIP_VERSION: 19.2.1
+      PIPENV_VERSION: 2018.10.13
+      SERVERLESS_VERSION: 1.48.4
+    cmd_test:
+      - pip --version
+      - python --version
+      - node --version
+      - npm --version
+      - yarn --version
+      - yarn run serverless --version
```

Comment:
When installed globally, you only need `serverless --version` or `sls --version`